HISTORY.rst

```diff
@@ -35,6 +35,10 @@ Unreleased Changes
 * Recreation
     * Validate values in the type column of predictor tables early in execution. Raise
       a ValueError if a type value isn't valid (leading/trailing whitespace is okay).
+* Validation
+    * Set a timeout on validation functions that open a file. This will raise an error to
+      prevent the UI from hanging if a file has to be downloaded from Google Drive File
+      Stream or a similar service.
```

Comment:
The original issue #261 has to do with file streaming services, but the approach we're taking here is more general and would apply to any file-based validation that takes too long.

And would it be worth mentioning that it'll raise a `warning` rather than an error?

Reply:
good point!